### DISCUSSION TITLE
Highlight/select grouped bars from slice

### DISCUSSION THREAD

**Discussion by matobiely** [plouc/nivo#1631] [Q&A] [answered]
Hi, i want to highlight or select all bars for particural x-tick and show tooltip with data from all bars. 
Here is example :
<img width="988" alt="image" src="https://user-images.githubusercontent.com/64771567/123400943-953ca880-d5a6-11eb-8c54-aecc7abbfa18.png">

Currently i can select/ highlight only one bar and show data in tooltip any for that one bar, but i want to show data from all bars and also highlight them all as one. So maybe if there is some possibility to select all bars as one slice, it will help.
I am using composed charts, Bar chart with custom line layer. Maybe try it in other way, Line chart with custom bar layer can solve it? 

Thanks

**Comment by wyze** [comment, marked as answer]
Can you provide a CodeSandbox with your current code? We can build from that. It should be possible with custom layer and tooltip.

**Reply by matobiely** [reply]
Here is codepen: https://codesandbox.io/s/bold-borg-8yxxh
For now ignore colors, main thing is to select slice of bar and show all data in tooltip

**Reply by wyze** [reply]
When you say `slice`, do you mean the group of 3 bars together? And to clarify the functionality, if you hover over any bar in that slice/group, you want them all to highlight and display a tooltip with the data for all 3 bars in that group?

**Reply by matobiely** [reply]
Yes, exactly like that

**Reply by wyze** [reply]
Cool, I think this is what you were looking for: https://codesandbox.io/s/vibrant-ives-3gkfx?file=/src/SliceLayer.js

**Reply by matobiely** [reply]
Just woow. Good job. Really. I didnt expected it so quickly. Thank you very much. 
Maybe you can consider to add this functionality directly to chart package? At least that one with `slice` i think, it could by helpful for others

**Reply by wyze** [reply]
Yeah I agree it would be a good addition. It looks like someone attempted a slice/group tooltip previously in #70, but it is really outdated. I'll see if I can include this in my typescript migration.
